cellmonitor/source/android/.gitignore

```diff
@@ -0,0 +1,116 @@
+##############################################################################
+## Copyright 2014-2016 CyberVision, Inc.
+## <p/>
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+## <p/>
+## http://www.apache.org/licenses/LICENSE-2.0
+## <p/>
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##############################################################################
+
+# Created by https://www.gitignore.io/api/android,intellij
+
+### Android ###
+# Built application files
+*.apk
```

Comment:
Are you sure that this .gitignore file should be here? Maybe one file for all projects is better and easier to maintain?